data/json/items/gunmod/underbarrel.json

```diff
@@ -592,5 +592,27 @@
     "location": "underbarrel",
     "mod_targets": [ "smg", "pistol" ],
     "min_skills": [ [ "weapon", 3 ], [ "gun", 1 ] ]
+  },
+  {
+    "id": "ree_33_tripod",
+    "type": "GUNMOD",
+    "name": { "str": "heavy-duty tripod" },
+    "description": "Massive tripod, used by exodii naval rifle so it can be used by lightweight units, providing firearm support.  Allow to use naval rifle while prone.",
```

Comment:
```suggestion
    "description": "Massive tripod, used by Exodii naval rifle so it can be used by lightweight units, providing firearm support.  Allows using naval rifle while prone.",
```